services/app-api/prisma/schema.prisma

```diff
@@ -20,18 +20,20 @@ model TestUser {
 
 // The StateSubmission Form Data
 model StateSubmission {
-  id                  String                    @id
-  state               State                     @relation(fields: [stateCode], references: [stateCode])
-  stateCode           String
-  revisions           StateSubmissionRevision[]
+  id        String                    @id
+  state     State                     @relation(fields: [stateCode], references: [stateCode])
+  stateCode String
+  revisions StateSubmissionRevision[]
 }
 
 model StateSubmissionRevision {
   id                  String          @id
-  createdAt           DateTime        
+  createdAt           DateTime
   submission          StateSubmission @relation(fields: [submissionID], references: [id])
   submissionID        String
   submissionFormProto Bytes
+
+  submittedAt DateTime?
```

Comment:
Is this `submittedAt` just the initial submission date? If so, I think we should called it that.  Or is it the date for the last time a submission was submitted by a state user and  thus re-locked?

Reply:
This is the submittedAt for a revision. I’ll rename the initiallySubmittedAt for the overall package.

Reply:
should we just call this `unlockedAt` then? It sounds like submission is really only relevant the first time and that is stored on the overall package.  after that we are just locking and unlocking revisions.